frontend/src/pages/LoginEmailFinished.tsx

```diff
@@ -33,7 +33,7 @@ const LoginEmailFinished = () => {
             window.localStorage.removeItem("emailForSignIn");
             setFinished(true);
           } catch (e: any) {
-            setError(e);
+            setError(e + "");
```

Comment:
So if something goes wrong here, it throws a null? Maybe if it's a null, it might be good to add a slightly more descriptive error to show to the user? Otherwise it'll just show an alert w/ null.